router/handler_edge_ctrl/hello.go

```diff
@@ -79,25 +81,23 @@ func (h *helloHandler) HandleReceive(msg *channel.Message, ch channel.Channel) {
 		if err := proto.Unmarshal(msg.Body, serverHello); err == nil {
 			pfxlog.Logger().Info("received server hello, replying")
 
-			byteData := serverHello.ByteData[edge_ctrl_pb.SignerPublicCertsHeader]
-
-			if byteData != nil {
-				signerCerts := &edge_ctrl_pb.SignerCerts{}
-				if err := proto.Unmarshal(byteData, signerCerts); err == nil {
-					h.stateManager.AddSignerPublicCert(signerCerts.Keys)
-				} else {
-					pfxlog.Logger().WithError(err).Error("could not unmarshal public key signers")
-				}
-			}
-
 			clientHello := &edge_ctrl_pb.ClientHello{
-				Version:   build.GetBuildInfo().Version(),
-				Listeners: h.listeners,
-
+				Version:       build.GetBuildInfo().Version(),
+				Listeners:     h.listeners,
 				Hostname:      h.hostname,
 				Protocols:     h.supportedProtocols,
 				ProtocolPorts: h.protocolPorts,
+				Data:          map[string]string{},
+			}
+
+			if serverHello.Data[common.DataRouterModel] == "true" {
+				clientHello.Data[common.DataRouterModel] = "true"
```

Comment:
Make headers. Not data.

Reply:
There are also capabilities added by Paul.

- The router sends a string array of capabilities.
- A bitmask that is sent from the controller